lib/sensors/led_driver.rs

```diff
@@ -0,0 +1,23 @@
+use hyped_io::i2c::{HypedI2c, I2cError};
+
+pub struct LedDriver<T: HypedI2c> {
+    i2c: T,
+    device_address: u8,
+}
+
+impl<T: HypedI2c> LedDriver<T> {
+    /// Create new instance of LED driver and attempt to configure
+    pub fn initialise(mut i2c: T, device_address: ledConfigRegister) -> Result<Self, ledDriverError> {
+        
+    }
+}
+
+
+
+const defaultLedDriverAddress: u8 = 0x30;
```

Comment:
The convention for Rust variable names is `snake_case` for normal variables and `SHOUTING_CASE` for constants (like these)